internal/db/db.go

```diff
@@ -0,0 +1,77 @@
+package db
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/jackc/pgtype"
+	pgtypeuuid "github.com/jackc/pgtype/ext/gofrs-uuid"
+	"github.com/jackc/pgx/v4"
+	"github.com/jackc/pgx/v4/pgxpool"
+)
+
+type DB struct {
+	Pool *pgxpool.Pool
```

Comment:
maybe add a reference to the logger here too so we can use it in the db package?